colors/dracula.vim

```diff
@@ -373,6 +373,628 @@ hi! link helpCommand DraculaPurple
+    hi! link @number Number
+    hi! link @number.float Float
+    " # Variable
+    hi! link @variable DraculaFg
+    hi! link @variable.builtin DraculaPurpleItalic
+    hi! link @variable.parameter DraculaOrangeItalic
+    hi! link @variable.member  DraculaOrange
+    " # Text
+    hi! link @text DraculaFg
+    hi! link @text.strong DraculaFgBold
+    hi! link @text.emphasis DraculaFg
+    hi! link @text.underline Underlined
+    hi! link @text.title DraculaYellow
+    hi! link @text.literal DraculaYellow
+    hi! link @text.uri DraculaYellow
+    hi! link @text.diff.add DiffAdd
+    hi! link @text.diff.delete DiffDelete
+
+    hi! link @markup.strong DraculaFgBold
+    hi! link @markup.italic DraculaFgItalic
+    hi! link @markup.strikethrough DraculaFgStrikethrough
+    hi! link @markup.underline Underlined
+
+    hi! link @markup Special
+    hi! link @markup.heading DraculaYellow
+    hi! link @markup.link Underlined
+    hi! link @markup.link.uri DraculaYellow
+    hi! link @markup.link.label SpecialChar
+    hi! link @markup.raw DraculaYellow
+    hi! link @markup.list Special
+
+    hi! link @comment Comment
+    hi! link @comment.error DiagnosticError
+    hi! link @comment.warning DiagnosticWarn
+    hi! link @comment.note DiagnosticInfo
+    hi! link @comment.todo Todo
+
+    hi! link @diff.plus Added
+    hi! link @diff.minus Removed
+    hi! link @diff.delta Changed
+
+    " # Tags
+    hi! link @tag DraculaCyan
+    hi! link @tag.delimiter DraculaFg
+    " HTML and JSX tag attributes. By default, this group is linked to TSProperty,
+    " which in turn links to Identifer (white).
+    hi! link @tag.attribute DraculaGreenItalic
+  endif
+  " }}}
+
+  " hrsh7th/nvim-cmp {{{
+  hi! link CmpItemAbbrDeprecated DraculaError
+
+  hi! link CmpItemAbbrMatch DraculaCyan
+  hi! link CmpItemAbbrMatchFuzzy DraculaCyan
+
+  hi! link CmpItemKindText DraculaFg
+  hi! link CmpItemKindMethod Function
+  hi! link CmpItemKindFunction Function
+  hi! link CmpItemKindConstructor DraculaCyan
+  hi! link CmpItemKindField DraculaOrange
+  hi! link CmpItemKindVariable DraculaPurpleItalic
+  hi! link CmpItemKindClass DraculaCyan
+  hi! link CmpItemKindInterface DraculaCyan
+  hi! link CmpItemKindModule DraculaYellow
+  hi! link CmpItemKindProperty DraculaPink
+  hi! link CmpItemKindUnit DraculaFg
+  hi! link CmpItemKindValue DraculaYellow
+  hi! link CmpItemKindEnum DraculaPink
+  hi! link CmpItemKindKeyword DraculaPink
+  hi! link CmpItemKindSnippet DraculaFg
+  hi! link CmpItemKindColor DraculaYellow
+  hi! link CmpItemKindFile DraculaYellow
+  hi! link CmpItemKindReference DraculaOrange
+  hi! link CmpItemKindFolder DraculaYellow
+  hi! link CmpItemKindEnumMember DraculaPurple
+  hi! link CmpItemKindConstant DraculaPurple
+  hi! link CmpItemKindStruct DraculaPink
+  hi! link CmpItemKindEvent DraculaFg
+  hi! link CmpItemKindOperator DraculaPink
+  hi! link CmpItemKindTypeParameter DraculaCyan
+
+  hi! link CmpItemMenu Comment
+  " }}}
+  
```

Comment:
When I check this out locally, it looks like there might be some trailing spaces here.

Reply:
Removed, thanks.